src/Confluent.RestClient/IConfluentClientSettings.cs

```diff
@@ -3,5 +3,6 @@
     public interface IConfluentClientSettings
     {
         string KafkaBaseUrl { get; }
+        long RequestTimeout { get; }
```

Comment:
I think it is more common to have time thresholds msec.
In Kafka for example, they use ".ms" or ".seconds" 
What do you think about renaming the property to "RequestTimeoutInMsec"?